src/main/resources/ipaas.yaml

```diff
@@ -12,7 +12,7 @@ defaultBuildParameters:
     deploy '
 
 builds:
-- name: atlasmap-2.2.0-${version.fuse.prefix}
+- name: atlasmap-2.2.1-${version.fuse.prefix}
```

Comment:
Need to update L63 of this file too.

And need to update repository fuse-jenkins-pipelines as well. See:
* https://github.com/jboss-fuse/fuse-jenkins-pipelines/pull/684
* https://github.com/jboss-fuse/fuse-jenkins-pipelines/pull/685

Reply:
Good Catch!